site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -154,9 +147,55 @@ API callers can't access group membership when users browse in Incognito mode, a
 removed when users clear their site data.
 
 
-{: #why}
+## How can I opt out of FLEDGE? {: #opt-out}
+
+You can block access to the FLEDGE API either as a site owner, or as an individual user.
+
+### How can sites control access? {: #opt-out-site}
+
+FLEDGE will eventually require sites to set permissions policy to allow FLEDGE functionality to be
+available. This will help ensure that arbitrary third parties can't use the API without a site's
+knowledge. However, to facilitate testing during [the first origin trial](/blog/privacy-sandbox-unified-origin-trial),
+this requirement is being [waived by default](https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md#permissions-policy).
+Sites that would like to explicitly disable FLEDGE functionality during the testing period can use
+the relevant permissions policy to block access.
+
+There are two FLEDGE permissions policies that can be set independently:
+* `join-ad-interest-group` enables/disables functionality to add a browser to interest groups
+* `run-ad-auction` enables/disables functionality to run an on-device auction
+
+Access to FLEDGE APIs can be disabled completely in first-party contexts by specifying the following
+permissions policy in an HTTP response header:
```

Comment:
```suggestion
`Permissions-Policy` in an HTTP response header:
```

Reply:
I could be convinced styling this copy as code is unnecessary. The logic was that this reference is about the HTTP header specifically.

Reply:
Good catch!